repository/models.go

```diff
@@ -1,24 +1,191 @@
 package repository
 
 import (
+	"attendance/util"
+	"encoding/json"
+	"net/http"
+	"os"
 	"time"
 
-	"github.com/google/uuid"
+	"github.com/go-pg/pg"
+	"go.uber.org/zap"
 )
 
 type User struct {
-	UserID   uuid.UUID
 	Username string `json:"username"`
-	Password string `json:"password"`
+	Password string `json:"password,omitempty"`
 	FullName string `json:"fullname"`
 	Class    int    `json:"class"`
-	Email    string `json:"email"`
+	Email    string `json:"email,omitempty"`
 	Role     string `json:"role"`
 }
 
 type Attendance struct {
-	UserID       uuid.UUID
```

Comment:
We are using POSTGRESQL , SQL Database , we could have used integers, saves cost in querying, mostly done in NOSQL,as we dont have set schema there.

Reply:
can be done later. Dont pick it up